src/ansys/fluent/core/quantity/dimensions.py

```diff
@@ -0,0 +1,123 @@
+import ansys.fluent.core.quantity as q
+
+
+class Dimensions(object):
+    """Initialize a Dimensions object using a dimensions list or a unit string.
+
+    Parameters
+    ----------
+    units: str
+        Unit string of quantity.
+    dimensions: list
+        List of dimensions.
+
+    Returns
+    -------
+    Dimensions instance.
+    """
+
+    def __init__(
+        self, units: str = None, dimensions: list = None, unit_sys: str = None
```

Comment:
Since it's supposed to be either `units` _or_ `dimensions`, the code should should check that both or not set.